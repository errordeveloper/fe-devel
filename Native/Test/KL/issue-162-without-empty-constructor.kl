struct MyType {
  Integer n;
  Scalar x;
};

operator entry() {
  MyType myType;
  report(myType);
}
