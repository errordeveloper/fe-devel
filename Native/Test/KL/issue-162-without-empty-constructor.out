{n:0,x:0}
